Jenkinsfile_CNP

```diff
@@ -31,7 +31,9 @@ def secrets = [
     secret('webapp-redis-connection-string', 'REDISCLOUD_URL'),
     secret('test-email', 'TEST_EMAIL'),
     secret('test-password', 'TEST_PASSWORD'),
-    secret('appinsights-instrumentationkey-mc', 'APPINSIGHTS_INSTRUMENTATIONKEY')
+    secret('appinsights-instrumentationkey-mc', 'APPINSIGHTS_INSTRUMENTATIONKEY'),
+    secret('system-user-name', 'SYSTEM_USER_NAME'),
```

Comment:
Which system? Are these generic credentials that will work across the board (which seems a bit risky) or are they for a particular service? If the latter, a more descriptive name would probably be better. The same applies to the password, below.

Reply:
we want it to be generic. This is a system user name which we are going to use it for several roles/services.